src/web/components/SignInGate/gateDesigns/SignInGatePatientia.tsx

```diff
@@ -0,0 +1,235 @@
+        color: ${palette.text.primary};
+
+        :hover {
+            border-bottom: none;
+        }
+    }
+`;
+
+const headingStyles = css`
+    ${headline.small()};
+    padding-bottom: ${space[6]}px;
+
+    ${from.phablet} {
+        ${headline.medium()};
+    }
+`;
+
+const subHeader = css`
+    ${body.medium({ fontWeight: 'bold' })};
+    border-top: 1px ${palette.line.primary} solid;
+    border-bottom: 1px ${palette.line.primary} solid;
+
+    padding: ${space[4]}px 0px;
+
+    color: ${palette.text.anchorPrimary};
+`;
+
+const bodyText = css`
+    ${body.medium()}
+    padding-top: ${space[2]}px;
+    padding-bottom: ${space[6]}px;
+    line-height: 135%;
+`;
+
+const signInBodyText = css`
+    padding-bottom: 0;
+    ${textSans.medium()}
+`;
+
+const actionButtons = css`
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    margin-bottom: ${space[6]}px;
+
+    > a {
+        margin-right: ${space[6]}px;
+    }
+`;
+
+const registerButton = css`
+    color: ${palette.text.ctaPrimary} !important;
+    text-decoration: none !important;
+`;
+
+const signInLink = css`
+    display: inline-block !important;
+`;
+
+const faq = css`
+    padding-top: ${space[3]}px;
+    padding-bottom: 18px;
+    margin-top: ${space[3]}px;
+
+    & a {
+        ${textSans.medium({ fontWeight: 'bold' })}
+        color: ${palette.text.primary};
+        display: block;
+        padding-bottom: ${space[3]}px;
+    }
+
+    & a:hover {
+        color: ${palette.text.primary};
+    }
+`;
+
+const firstParagraphOverlay = (isComment: boolean) => css`
+    margin-top: -250px;
+    width: 100%;
+    height: 250px;
+    position: absolute;
+
+    /* "transparent" only works here because == rgba(0,0,0,0) */
+    background-image: linear-gradient(
+        0deg,
+        ${isComment ? opinion[800] : palette.background.primary},
+        70%,
+        rgba(255, 255, 255, 0)
+    );
+`;
+
+// This css hides all the elements in the article after the #sign-in-gate
+// using the General sibling combinator https://developer.mozilla.org/en-US/docs/Web/CSS/General_sibling_combinator
+const hideElementsCss = `
+    #sign-in-gate ~ * {
+        display: none;
+    }
+    `;
+
+// set the ophan component tracking vars
+export const withComponentId: (id: string) => OphanComponent = (
+    id: string = '',
+) => ({
+    componentType: 'SIGN_IN_GATE',
+    id,
+});
+
+export const SignInGatePatientia = ({
+    signInUrl,
+    guUrl,
+    dismissGate,
+    abTest,
+    component,
+    isComment,
+}: SignInGateProps) => {
+    const [showCpmUi, setShowCmpUi] = useState(false);
+
+    return (
+        <div className={cx(signinGate)} data-cy="sign-in-gate-patientia">
+            <style>{hideElementsCss}</style>
+            <div className={cx(firstParagraphOverlay(!!isComment))} />
+            <h1 className={cx(headingStyles)}>
+                Register for free and continue reading
+            </h1>
+            <h2 className={cx(subHeader)}>
```

Comment:
This is intentional, since it better reflects the designs. In one design (`SignInGateMain`) this text is a bold version of the body text, therefore used a `p` tag. But in the other design (`SignInGatePatientia`) this was more of a sub-heading text, and therefore used the `h2` tag in that design.